### PR TITLE
Add cljs.test

### PR DESCRIPTION
**Pull request by borkdude**

### CODE REVIEW

**Comment by borkdude** [comment]
Actually this only works with `(require '[cljs.test])` right now and not with `(require '[clojure.test])`. I'm going to make something in SCI so you can configure some namespace aliases as a follow PR on this.

**Review by PEZ — COMMENTED** [review]
The changes LGTM!

Update the Changelog, api docs and add some example, maybe?

**Comment by borkdude** [comment]
I also took the liberty to rename `life-cycle` to `lifecycle` which I think the most common spelling of that word.

**Comment by borkdude** [comment]
Oh, I was wrong about that, haha, but they are both correct and I prefer less hyphens in namespaces, which is maybe silly, so feel free to change it back if it bothers you.

File: src/joyride/lifecycle.cljs [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+(defn maybe-run-init-script+ [run-fn {:keys [label script script-path script-abs-path]}]
+  (utils/say (str label " script: " script-path))
+  (-> (utils/path-or-uri-exists?+ script-abs-path)
+      (p/then (fn [exists?]
+                (if exists?
+                  (do
+                    (utils/sayln (str ".  Running..."))
+                    (run-fn script))
+                  (utils/sayln (str ".  No " label " script present")))))))
```
Comment (PEZ) [inline src/joyride/lifecycle.cljs:38]: I think this file must have been created by mistake. I'll remove it.

**Pull request merged**